src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -143,6 +173,9 @@ PIC_Ions
 typedef MakeSeq<
 Species1,
 Species2
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
```

Comment:
*offline discussion with @Heikman and @Flamefire:*
Please check if the list
```c++
typedef typename PMacc::particles::traits::FilterByFlag<VectorAllSpecies,
+                                                                synchrotronPhotons<> >::type AllSynchrotronPhotonsSpecies;
```
is empty or not. If it is not empty, add photons.